solarsystem.rb

```diff
@@ -0,0 +1,156 @@
+
+
+class SolarSystem
+  attr_accessor :planets,
+                :formation_yr
+
+  def initialize(planets, formation_yr)
+    @planets = planets
+    @formation_yr = formation_yr
+  end
+
+  def add_planet(planet)
+    @planets.push(planet)
+  end
+
+  def add_planets(planets)
+    @planets = @planets + (planets)
+  end
+
+  def info
+    @planets.each do |planet|
+      puts planet.info
+      puts
+    end
+  end
+end
+
+class Planet
+  attr_accessor :name,
+                :mass,
+                :diameter,
+                :moon,
+                :average_temp,
+                :distance_from_the_sun,
+                :num_from_the_sun
+
+  def initialize(attrs)
+
+    @name = attrs[:name]
+    @mass = attrs[:mass]
+    @diameter = attrs[:diameter]
+    @moon = attrs[:moon]
+    @average_temp = attrs[:average_temp]
+    @distance_from_the_sun = attrs[:distance_from_the_sun]
+    # if attrs[:num_from_the_sun] != ""
+    #   attrs[:num_from_the_sun] = attrs[:num_from_the_sun] + " "
+    # end
+    @num_from_the_sun = attrs[:num_from_the_sun]
+  end
+
+  def info
+    return "#{@name} has a mass of #{@mass} and has a diameter of #{@diameter}. It has #{@moon} moon(s), and has an average temperature of #{@average_temp}. #{@name} is #{@distance_from_the_sun} from the sun, making it the #{@num_from_the_sun}closest planet to the sun."
```

Comment:
Just fyi, `return` is not needed here, it's fine if it is, but if it weren't it would not change the output.